Realm/RLMUtil.mm

```diff
@@ -360,6 +360,15 @@ BOOL RLMIsRunningInPlayground() {
     return [[NSBundle mainBundle].bundleIdentifier hasPrefix:@"com.apple.dt.playground."];
 }
 
+void RLMWorkaroundRadar31252694() {
+    SEL isNSArray = NSSelectorFromString([@[@"is", @"NSArray", @"_", @"_"] componentsJoinedByString:@""]);
+    IMP yes = imp_implementationWithBlock(^(__unused id array) { return YES; });
+    Method isInvalidated = class_getInstanceMethod(RLMArray.class, @selector(isInvalidated));
+    const char *typeEncoding = method_getTypeEncoding(isInvalidated);
+    class_addMethod(RLMArray.class, isNSArray, yes, typeEncoding);
+    class_addMethod(RLMResults.class, isNSArray, yes, typeEncoding);
```

Comment:
To cover the RealmSwift classes I think we'd need to do the same on `RLMListBase`, `RealmSwift.Results` and `RealmSwift.LinkingObjects`. `RLMListBase` is defined by Realm.farmework and so should always be available, while the other two are only available if RealmSwift is loaded and would need to be looked up via `NSClassFromString`.